src/store/rule/types.ts

```diff
@@ -1,11 +1,13 @@
 import {Group} from "../group/types";
 
 export interface Rule {
-  ID?: string
-  Name: string
-  Source: Group[] | string[] | null
-  Destination: Group[] | string[] | null
-  Flow: string
+    ID?: string
+    Name: string
+    Description: string
```

Comment:
Codacy has a fix for the issue: [Replace `····Description:·string` with `··Description:·string;`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  Description: string;
```